src/components/giga-packet/gigaPacket.tsx

```diff
@@ -317,6 +327,14 @@ export function GigaPacket() {
 
             setLoadingStates('loading')
 
+            let totalTxs = 0
+            _formState.forEach((item) => {
+                const x = _utils.divideAndRemainder(item.numberOfSlots)
```

Comment:
This can't bug out? What if x is undefined? Don't want the display code to throw an error and bug out the whole process

Reply:
x should never be undefined, item.numberofslots is checked above in checkForm